meson.build

```diff
@@ -0,0 +1,75 @@
+# Meson build script from MSFITSLIB
+project('msfitslib', 'cpp', version: '1.0.1',
+         default_options : ['cpp_std=c++11',])
+
+# Basic setup
+cc  = meson.get_compiler('cpp')
+builddir = meson.project_build_root()
+
+comp_flags = ['-D_UNIX','-D_ENABLE_PROFILER_', '-D_ENABLE_TRACING_', 
+              '-D_ENABLE_VISUALIZATION_', '-D_MONTE_CARLO_', '-D_NO_ROOT_']
+add_project_arguments(comp_flags, language : 'cpp')
+
+# Load list of files to compile into shared object
+# Note: Meson does not allow * wildcard (deliberately)
+# Instead, we use file generated from command: ls src/*.cpp > meson_srclist.txt
+fs = import('fs') 
+srcs = fs.read('meson_srclist.txt').strip().split('\n')
+
+# Dependencies: FFTW3, CFITSIO, LIBNOVA, LDL, PTHREAD
+fftw3_dep = dependency('fftw3')
+cfitsio_dep = dependency('cfitsio')
+libnova_dep = cc.find_library('libnova', dirs : '/usr/lib/x86_64-linux-gnu') 
+ldl_dep     = cc.find_library('dl') 
+lpthread_dep = cc.find_library('pthread')
```

Comment:
In general, you should prefer to find threads with `dependency('threads')` -- this will usually pass the `-pthread` flag on GCC, rather than `-lpthread`, and the former is considered superior (it can handle more than just linking the library in).

Reply:
Easy change, thanks